examples/README.md

```diff
@@ -106,3 +106,111 @@ MOMENTO_AUTH_TOKEN=<YOUR AUTH TOKEN> npm run load-gen
 
 You can check out the example code in [load-gen.ts](load-gen.ts). The configurable
 settings are at the bottom of the file.
+
+## Running the request-coalescing example
+
+This repo includes a request-coalescer, to allow you to experiment
+with performance in your environment based on different configurations.
+
+The request-coalescing example shows difference between latencies when
+a basic load-generator is used v/s when request-coalescer is used.
+
+Check out the configuration settings at the bottom of the 'request-coalescing.ts' to
+see how different configurations impact performance.
+
+Here are some stats and steps on how to run the example:
+Change the configuration found at the end of the file as follows to achieve the stats shown below:
+
+| Configuration Name         | Value |
+|----------------------------|-------|
+| maxRequestsPerSecond       | 1000  |
+| numberOfConcurrentRequests | 1000  |
+
+Stats should like:
+- Before Request-Coalescing:
+
+```javascript
+[2023-03-16T22:01:01.787Z] INFO (Momento: request-coalescer-load-gen):
+cumulative stats:
+  total requests: 60000 (997 tps, limited to 1000 tps)
+success: 60000 (100%) (997 tps)
+unavailable: 0 (0%)
+deadline exceeded: 0 (0%)
+resource exhausted: 0 (0%)
+rst stream: 0 (0%)
+
+cumulative set latencies:
+
+  count: 30000
+min: 18
+p50: 60
+p90: 161
+p99: 445
+p99.9: 544
+max: 561.454333
+
+
+cumulative get latencies:
+
+  count: 30000
+min: 18
+p50: 40
+p90: 95
+p99: 140
+p99.9: 385
+max: 394.540167
+```
+
+- After Request-Coalescing:
+
+```javascript
+[2023-03-16T22:02:02.338Z] INFO (Momento: request-coalescer-load-gen):
+cumulative stats:
+total requests: 60000 (999 tps, limited to 1000 tps)
+       success: 60000 (100%) (999 tps)
+   unavailable: 0 (0%)
+deadline exceeded: 0 (0%)
+resource exhausted: 0 (0%)
+    rst stream: 0 (0%)
+
+cumulative set latencies:
+
+  count: 30000
+    min: 8
+    p50: 20
+    p90: 26
+    p99: 37
+  p99.9: 40
+    max: 44.164916
+
+
+cumulative get latencies:
+
+  count: 30000
+    min: 8
+    p50: 20
+    p90: 26
+    p99: 32
+  p99.9: 36
+    max: 39.292083
+
+
+[2023-03-16T22:02:02.338Z] INFO (Momento: request-coalescer-load-gen):
+For request coalescer:
+Number of set requests coalesced:
+29700 (49.5%)
+
+Number of get requests coalesced:
+29700 (49.5%)
+```
+Notice the reduction in cumulative latency foe get/set requests after coalescing.
+
+### Run the above example:
+```bash
+# Run example load generator
+MOMENTO_AUTH_TOKEN=<YOUR AUTH TOKEN> npm run load-gen
+```
+
+### More stats with different configurations:
+You can find more stats with different configurations
+[here](https://momentohq.notion.site/Request-Coalescing-Stats-7c4efeeab9d448538647712e3d7e1dff)
```

Comment:
if you want to include the rest of the stats that's totally fine, but you will want to do it by adding another Markdown file to the repo, not linking to notion.  Up to you whether you want to do this or not, it's fine to omit it for now if you like.